include/core/binary_reader.hpp

```diff
@@ -23,12 +23,12 @@ namespace rive
 
         size_t lengthInBytes() const;
 
-        uint64_t readVarUint();
         std::string readString();
         double readFloat64();
         float readFloat32();
         uint8_t readByte();
```

Comment:
For consistency, I thought of changing this to `readUint8()` but I felt that it was out of the scope of this PR.

Reply:
That's fair, I'd probably call it readVarUint8 as it's variable length encoded LEB128.